applications/asset_tracker_v2/overlay-memfault.conf

```diff
@@ -24,3 +24,7 @@ CONFIG_DEBUG_MODULE=y
 # Increase the event storage size so that all metrics generated by the asset tracker application
 # are reliably sent to the memfault cloud.
 CONFIG_MEMFAULT_EVENT_STORAGE_SIZE=2048
+CONFIG_NRF_MODEM_LIB_TRACE=y
```

Comment:
Perhaps we should make a Kconfig option that select these options. `CONFIG_DEBUG_MODULE_MEMFAULT_SEND_MODEM_TRACES_ON_CRASH` (very long, but something similar).

The user can then easily opt out of capturing modem traces by disabling a single option.